examples/fitting_NuSTAR_spectra-duncan2021.py

```diff
@@ -0,0 +1,173 @@
+spec.params["EM1_spectrum1"] = {"Value": 14, "Bounds": (1e0, 1e2)}
+spec.params["T2_spectrum1"] = {"Value": 10, "Bounds": (5, 15)}
+spec.params["EM2_spectrum1"] = {"Value": 0.46, "Bounds": (1e-4, 10)}
+spec.params["C_spectrum2"] = {"Status": "free", "Bounds": (0.5, 2)}
+
+#####################################################
+#
+# Fit lower energy range with the first thermal model first
+spec.params["T2_spectrum1"] = "frozen"
+spec.params["EM2_spectrum1"] = "frozen"
+spec.energy_fitting_range = [2.5, 4]
+
+spec.fit(tol=1e-6)
+print(spec.params)
+print(spec.rParams)
+
+#####################################################
+#
+# Now fit higher energy range with the second thermal model
+
+spec.params["T1_spectrum1"] = "frozen"
+spec.params["EM1_spectrum1"] = "frozen"
+spec.params["C_spectrum2"] = "frozen"
+spec.params["T2_spectrum1"] = "free"
+spec.params["EM2_spectrum1"] = "free"
+
+#####################################################
+#
+# Need the gain slope to vary too for this microflare but only needed for the 6.7 keV line
+print(spec.rParams)
+spec.rParams["gain_slope_spectrum1"] = "free"
+spec.rParams["gain_slope_spectrum2"] = spec.rParams["gain_slope_spectrum1"]
+
+spec.energy_fitting_range = [4, 10.8]
+
+# print(spec.rParams)
+
+spec.fit(tol=1e-6)
+print(spec.params)
+print(spec.rParams)
+
+#####################################################
+#
+# Now free everything over full range
+spec.params["T1_spectrum1"] = "free"
+spec.params["EM1_spectrum1"] = "free"
+spec.params["C_spectrum2"] = "free"
+
+spec.energy_fitting_range = [3, 10.8]
+
+spec.fit(tol=1e-10)
+print(spec.params)
+print(spec.rParams)
+
+#####################################################
+#
+# plot the result
+plt.rcParams["font.size"] = spec_font_size
+plt.figure(figsize=spec_plot_size)
+axes, res_axes = spec.plot()
+for a in axes:
+    a.set_xlim(dunc_xlims)
+    a.set_ylim(dunc_ylims)
+plt.show()
+plt.rcParams["font.size"] = default_font_size
+
+#####################################################
+#
+# For the 2 thermal model fitting:
+#
+# | Model Parameter                | XSPEC (Duncan et al. 2021)         | This Work                       |
+# | :---                           |    :----:                          |                            ---: |
+# | Temperature 1 [MK]             | 4.1$^{+0.2}_{-0.1}$                | 4.8$\pm$0.3                   |
+# | Emission Measure 1 [cm$^{-3}$] | 1.4$^{+0.6}_{-0.4}\times$10$^{47}$ | 7.6$\pm$2.4$\times$10$^{46}$  |
+# | Temperature 2 [MK]             | 10.00$^{+0.03}_{-0.03}$            | 10.39$\pm$0.09                 |
+# | Emission Measure 2 [cm$^{-3}$] | 4.6$^{+0.1}_{-0.2}\times$10$^{45}$ | 4.3$\pm$0.3$\times$10$^{45}$  |
+#
+#
+# | Response Parameter             | XSPEC (Duncan et al. 2021)         | This Work         |
+# | :---                           |    :----:                          |              ---: |
+# | Gain Slope                     | 0.977$\pm$0.002                    | 0.978$\pm$0.001   |
```

Comment:
So these don't format as table in the rendered doc, you could rewrite this as [rst table](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#tables) or something else.